frontend/src/components/BlockGraph.tsx

```diff
@@ -0,0 +1,156 @@
+import { RefObject, ReactNode } from 'react';
+import classNames from 'classnames';
+import css from './BlockGraph.module.scss';
+
+const classes = classNames.bind(css);
+
+const lerp = (values: number[], lo: number, hi: number) => {
```

Comment:
I'd add a bit more descriptive names for the `hi` and `lo` to improve readability

Reply:
naming is hard, spelt out the names now, but I'm open for suggestions if there are some clearly better ones

Reply:
Yeah, `mapToRange` might be a better name

as a side note: we "could" use `map`, it's just a method name, but that would be confusing